README.md

```diff
@@ -39,15 +38,27 @@ A Polkadot SDK based project such as this one consists of:
 ```
 cargo install --force --locked pop-cli
 ```
-
 #### Start a new parachain project with this template:
 ```
 pop new parachain
 ```
 
 #### Learn how to run your parachain using the `pop up` command:
 ```sh
-pop up parachain -f ./network.toml
+# Generate the chain spec
+pop build spec            \
```

Comment:
@AlexD10S I'll keep this PR as a draft until I can actually make this command work.